testdata/BUILD

```diff
@@ -502,6 +502,22 @@ py_image(
     main = "py_image.py",
 )
 
+load("//python3:image.bzl", "py3_image")
+
+py_library(
+    name = "py3_image_library",
+    srcs = ["py3_image_library.py"],
+)
```

Comment:
Not sure this is meaningfully different, can we just use the existing `py_libary`?  LMK if I'm missing a change.

Reply:
Nope, this can be removed and the existing one reused.